synapse/lib/stormtypes.py

```diff
@@ -159,6 +159,32 @@ def addLibFuncs(self):
 
         return text
 
+class LibBytes(Lib):
+
+    def addLibFuncs(self):
+        self.locls.update({
+            'put': self._libBytesPut,
+        })
+
+    async def _libBytesPut(self, byts):
+        '''
+        Save the given bytes variable to the axon.
+
+        Returns:
+            ($size, $sha256)
+
+        Example:
+            ($size, $sha2) = $lib.bytes.put($bytes)
+        '''
+        if not isinstance(byts, bytes):
+            mesg = '$lib.bytes.put() requires a bytes argument'
+            raise s_exc.BadArg(mesg=mesg)
+
+        #TODO decide on access pattern
+        await self.runt.snap.core.axready.wait()
```

Comment:
We don't want to wait *forever*; at some point we need to timeout and raise an exception here.